docs/spec/api.md.tmpl

```diff
@@ -116,13 +116,20 @@ indicating what is different. Optionally, we may start marking parts of the
 specification to correspond with the versions enumerated here.
 
 <dl>
+	<dt>2.1</dt>
```

Comment:
So this version is the specification version, not the registry version. This should be 2.0.3.